commands_to_load/Here.py

```diff
@@ -0,0 +1,60 @@
+# Commands for finding who has access to certain channels.
+# Useful since the server size does not allow offline users to be listed
+#     in the sidebar
+
+from discord.ext import commands
+import discord
+import logging
+logger = logging.getLogger('wall_e')
+
+
+class Here():
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    def build_embed(members, channel):
+        # build response
+        embed = discord.Embed(type = "rich")
+        embed.title = "Users in **" + channel.name + "**"
+        embed.color = discord.Color.blurple()
+        embed.set_footer(text="brenfan", icon_url="https://i.imgur.com/vlpCuu2.jpg")
+        if len(members) == 0:
+            string = "I couldnt find anyone."
+        elif len(members) > 50:
+            string = "There's a lot of people here."
+        else:
+            string =  "The following users have permission for this channel.\n"
+            for member in members:
+                display = member.display_name + "   (" + str(member) + ")\n"
+                string += display
+
+        embed.description = string
+        return embed
+
+    @commands.command()
+    async def here(self, ctx, *search):
+        logger.info("[HereCommands here()] "
```

Comment:
should start as `logger.info("[Here here()] "` to match convention.

also please add some more logs, one doesnt really seem like enough.